test/instrumentation/index.test.js

```diff
@@ -617,7 +525,7 @@ test('nested spans', function (t) {
     t.strictEqual(s0.transaction_id, trans.id, 's0 transaction_id matches transaction id')
 
     const s1 = findObjInArray(data.spans, 'name', 's1')
-    t.strictEqual(s1.parent_id, trans.id, 's1 should directly descend from the transaction')
+    t.strictEqual(s1.parent_id, s0.id, 's1 should descend from s0')
```

Comment:
REVIEW NOTE: This is the intentional change in behaviour that sync span creation results in parent/child relationships, rather than siblings.

```
Expected hierarchy before:
  transaction "foo"
  `- span "s0"
    `- span "s01"
  `- span "s1"
    `- span "s11"
    `- span "s12"

After:
  transaction "foo"
  `- span "s0"
    `- span "s1"
      `- span "s11"
      `- span "s12"
    `- span "s01"
```

The change is that "s1" is a child of "s0". See discussion at https://github.com/elastic/apm-agent-nodejs/issues/1889